readthedocs/core/mixins.py

```diff
@@ -22,3 +25,29 @@ class LoginRequiredMixin:
     @method_decorator(login_required)
     def dispatch(self, *args, **kwargs):
         return super().dispatch(*args, **kwargs)
+
+
+class HideProtectedLevelMixin:
+
+    """
+    Hide ``protected`` privacy level from Form.
+
+    Remove Protected for now since it cause confusions to users.
+
+    If the current ``privacy_level`` is ``protected`` we show it (so users keep
+    seeing consistency values), and hide it otherwise (so it can't be selected).
+
+    There is a better way to manage this by using Version states See:
+    https://github.com/rtfd/readthedocs.org/issues/5321
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if any([
+            not self.instance,
+            self.instance and self.instance.privacy_level != PROTECTED,
+        ]):
```

Comment:
This logic feels quite complex. Can we not just do something like:

`if self.instance is None or self.instance.privacy_level != PROTECTED`